actors/builtin/power/power_state.go

```diff
@@ -146,10 +159,29 @@ func (st *State) addToClaim(s adt.Store, miner addr.Address, power abi.StoragePo
 		return errors.Errorf("no claim for actor %v", miner)
 	}
 
+	oldNominalPower, err := st.computeNominalPower(s, miner, claim.Power)
+	if err != nil {
+		return err
+	}
 	claim.Power = big.Add(claim.Power, power)
+	newNominalPower, err := st.computeNominalPower(s, miner, claim.Power)
+	if err != nil {
+		return err
+	}
+
```

Comment:
Prefer to move this after the following line so the claim pledge/power updates remain grouped.